dist/APIManager.js

```diff
@@ -0,0 +1,42 @@
+class APIManager {
+
+    getData(urlAdd, func, name, wrongData) {
+        $.ajax({
+            method: 'GET',
+            url: urlAdd,
+            success: (data) => {
+                if (data === 'invalid data') {
+                    wrongData();
```

Comment:
`wrongData` is not really to right name for this function if it's in the `success` property...